src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js

```diff
@@ -1,110 +1,151 @@
-
-                            chart.append('text')
-                                .attr('x', graphWidth - margin + 3)
-                                .attr('y', heigth / 2 + 5)
-                                .attr('text-anchor', 'start')
-                                .attr('font-size', 12)
-                                .attr('font-family', 'sans-serif')
-                                .text(average.toFixed(2));
+                                .attr('height', height);
 
                             chart.append('g')
                                 .attr('class', 'x axis')
-                                .attr('transform', 'translate(0,' + (heigth - margin) + ')')
+                                .attr('transform', 'translate(0,' + (height - margin) + ')')
                                 .call(d3.axisBottom(scaleX).tickFormat(function(d) {
                                     return dates[d];
                                 }));
-                            
-                            chart.append('g')
-                                .attr('class', 'y axis')
-                                .attr('transform', 'translate(' + margin + ', 0)')
-                                .call(d3.axisLeft(scaleY));
-
-                            var scaleYValue = (scaleY(average == 0 ? 1 : average) + 3) | 0;
-
-                            chart.append('text')
-                                .attr('x', 0)
-                                .attr('y', scaleYValue)
-                                .attr('text-anchor', 'start')
-                                .attr('font-size', 12)
-                                .attr('font-family', 'sans-serif')
-                                .text('ø');
+
+                            drawDataSeries(firstSeries,
+                                attrs.firstSeriesColor,
+                                true,
+                                scaleX,
+                                chart,
+                                height,
+                                graphWidth,
+                                margin,
+                                dates,
+                                'throughput');
+
+                            drawDataSeries(secondSeries,
+                                attrs.secondSeriesColor,
+                                false,
+                                scaleX,
+                                chart,
+                                height,
+                                graphWidth,
+                                margin,
+                                dates,
+                                'queue-length');
+
+                            chart.append("text")
+                                .attr("text-anchor", "middle")
+                                .attr("transform", "translate(" + (margin / 2) + "," + (height / 2) + ")rotate(-90)")
+                                .text("Throughput [msgs/s]");
```

Comment:
@WilliamBZA do we want to parametrize this value? At the moment label is hard coded, as well as a class name. I could add it to the data set passed to the graph, and populate the values in a controller. What do you think?

Reply:
I decided to do that